### ISSUE TITLE
unable to interact with nickserv

### ISSUE THREAD

**Issue by csylvain** [matrix-org/matrix-appservice-irc#1171]
using Element version: 1.7.10
Homeserver is https://matrix-client.matrix.org
Identity Server is https://vector.im
-----
joined a freenode irc channel which requires registered nick to send a message. 
attempting `/msg @freenode_Nickserv:matrix.org help`  or `/msg nickserv help` in @freenode_Chanserv:matrix.org channel produces only "Command Error" about the `/msg` command.
need to be able to issue `group` and `identify` messages to nickserv.

**Comment by Half-Shot** [comment]
You should try starting a dm with @freenode_NickServ:matrix.org from your matrix client.

**Comment by csylvain** [comment]
i did.     did i not indicate the "/msg" command was used?

**Comment by Legogris** [comment]
`@freenode_NickServ:matrix.org` seems to be having issues - it does not respond to any messages at all for me

**Comment by csylvain** [comment]
yeah .. i'm an old irc hand and i reported this issue only after having
tried repeatedly over the course of a week.   hexchat (irc app) interacts
with nickserv just fine.

**Comment by kodxana** [comment]
Having that same issue cant talk with nick server to change my name

**Comment by BenjamenMeyer** [comment]
having the same issue with `NickServ` and `ChanServ` for Libera IRC too - try `/msg NickServ <my channel>` or `/msg ChanServ <my channel>` and just get the below error message, and I'm in a DM with the respective users.
![image](https://user-images.githubusercontent.com/1074110/209188672-7359df7d-e5de-4f3a-bf5c-1eb3d72e3ada.png)

**Comment by sunmin89** [comment]
> having the same issue with `NickServ` and `ChanServ` for Libera IRC too - try `/msg NickServ <my channel>` or `/msg ChanServ <my channel>` and just get the below error message, and I'm in a DM with the respective users. ![image](https://user-images.githubusercontent.com/1074110/209188672-7359df7d-e5de-4f3a-bf5c-1eb3d72e3ada.png)

Matrix is not  the same as IRC, You can jump to  librechat's `NickServ`  channel through three steps.
![mm](https://user-images.githubusercontent.com/17847861/229976094-3a2e1808-2994-4d3a-934f-7012ba9f18ea.png)
